src/mozanalysis/frequentist_stats/sample_size.py

```diff
@@ -20,6 +23,344 @@
+        self, empirical_sizing_df: pd.DataFrame
+    ) -> Styler:
+        """Pretty-print the DF returned by `empirical_sizing()`.
+
+        Returns a pandas Styler object.
+        """
+        return (
+            empirical_sizing_df[
+                [
+                    "relative_effect_size",
+                    "population_percent_per_branch",
+                    "sample_size_per_branch",
+                    "effect_size_value",
+                    "mean_value",
+                    "std_dev_value",
+                ]
+            ]
+            .rename(
+                columns={
+                    "relative_effect_size": "rel_effect_size",
+                    "population_percent_per_branch": "branch_pop_pct",
+                    "sample_size_per_branch": "branch_sample_size",
+                }
+            )
+            .style.format(thousands=",", precision=4)
+            .format(subset="branch_sample_size", thousands=",", precision=0)
+            .format("{:.2f}%", subset="branch_pop_pct")
+            .format("{:.2%}", subset="rel_effect_size")
+        )
+
+    @property
+    def dataframe(self) -> pd.DataFrame:
+        """dataframe property
+
+        returns results consolidated into a dataframe"""
+
+        formatted_results = {}
+        for m, r in self.data.items():
+            metric_result = {}
+            for k, v in r.items():
+                if isinstance(v, dict):
+                    for vk, vv in v.items():
+                        if vk == "period_start_day":
+                            vk = "period"
+                        metric_result[f"{k}_{vk}"] = vv
+                else:
+                    metric_result[k] = v
+            formatted_results[m] = metric_result
+
+        df = pd.DataFrame.from_dict(formatted_results, orient="index")
+
+        return df[
+            [
+                "relative_effect_size",
+                "effect_size_value",
+                "mean_value",
+                "std_dev_value",
+                "sample_size_per_branch",
+                "population_percent_per_branch",
+                "effect_size_period",
+                "mean_period",
+                "std_dev_period",
+            ]
+        ]
+
+    def get_styled_dataframe(self) -> Styler:
+        """returns styled dataframe for results from
+        empirical_effect_size_sample_size_calc
+
+        Arguments:
+            style (bool): If true, return a cleaned up and formatted pandas Styler.
+            Otherwise return a dataframe
+        Returns:
+            Styler: styled dataframe for visualization
+        """
+        df = self.dataframe
+        return self.style_empirical_sizing_result(df)
+
+
+class SampleSizeCurveResultHolder(ResultsHolder):
+    def __init__(self, *args, **kwargs):
+        """
+        Args:
+            metrics (dict, optional): _description_. Defaults to None.
+            params (dict, optional): _description_. Defaults to None.
+        """
```

Comment:
remove?

Reply:
Yeah, replaced with a comment highlighting the use of the ResultsHolder __init__